app/cluster_discovery.go

```diff
@@ -33,20 +33,21 @@ func (a *App) NewClusterDiscoveryService() *ClusterDiscoveryService {
 
 func (me *ClusterDiscoveryService) Start() {
 
-	<-me.app.Srv.Store.ClusterDiscovery().Cleanup()
+	me.app.Srv.Store.ClusterDiscovery().Cleanup()
```

Comment:
Would you please check and log the returned error?

Reply:
sure

Reply:
Done! :)